components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -162,10 +163,10 @@
                         }
                     };
 
-                    setTimeout(openTreeFn, 1000, instance, this); 					
+                    setTimeout(openTreeFn, 1000, instance, this);
```

Comment:
Let's discuss this.  I see this is existing stuff but, would love to dig in to it a bit